hcc_config/hcc_config.cpp

```diff
@@ -166,6 +166,8 @@ void ldflags(void) {
     std::cout << " -L" XSTR(CODEXL_ACTIVITY_LOGGER_LIBRARY);
     std::cout << " -lCXLActivityLogger";
 #endif
+
+    std::cout << " -lhc_profile";
```

Comment:
@otauqir I'm not fond of the idea to add a mandatory library to be linked by every HCC application. Plus the implementation of `hc_profile` forces strong dependency to CodeXL library. I think it would be better to use dynamic shared library, and only load on demand.